packages/components/src/dashicon/index.js

```diff
@@ -913,6 +913,8 @@ export default class Dashicon extends Component {
 				width={ size }
 				height={ size }
 				viewBox="0 0 20 20"
+				style={ style }
+				{ ...color && { color } }
```

Comment:
Can this be just `{ ...props }` to avoid introducing a mobile specific prop to the web component?
The `SVG` is a component that have two different implementation, this is where the specific code should go right?

Reply:
Thank you @youknowriad for the review!
I updated to pass all remaining props directly to SVG. Tested on mobile (iOS and Android) and it's behaving properly. 🎉

Could you check again please?
cc @Tug

Reply:
Thanks for the update, I like this implementation better.